bin/sumd

```diff
@@ -172,16 +176,24 @@ def main():
 
     # Create task queue for worker threads, enable thread processing
     tasks = Queue.Queue(options.task_limit)
+    riemann_events = Queue.Queue(options.sender_limit)
```

Comment:
I imagine we want to let there be more than 1 thing in the Queue at once - https://docs.python.org/2/library/queue.html#Queue.Queue

Reply:
Also, to help with future readability, I recommend calling this `event_queue` instead.